src/System.Private.Xml/src/System/Xml/XmlQualifiedName.cs

```diff
@@ -10,11 +10,13 @@ namespace System.Xml
     /// <devdoc>
     ///    <para>[To be supplied.]</para>
     /// </devdoc>
+    [Serializable]
     public class XmlQualifiedName
     {
         private string _name;
         private string _ns;
 
+        [NonSerialized]
```

Comment:
It won't.  I'd need to modify it to handle that, which I expect would be mostly doable but not quite as easy, as we've renamed many of the fields.  We can explore doing so though.